docs/resources/aws_ec2_dhcp_options.md

```diff
@@ -0,0 +1,39 @@
+---
+title: About the aws_ec2_dhcp_options Resource
+platform: aws
+---
+
+# aws\_ec2\_dhcp\_options
+
+Use the `aws_ec2_dhcp_options` InSpec audit resource to test properties of a single AWS DHCP Options.
```

Comment:
Is this testing one set of options, or multiple sets of options?

Reply:
Multiple Set of Options actually!